scripts/src/main/resources/scripts/functions

```diff
@@ -1824,53 +1830,142 @@ function doGetSoftwareVersion() {
   fi
 }
 
+# $1: software
+function doGetSoftwareEdition() {
+  local software="${1}"
+  local edition
+  echo "${software}" | grep '-' >/dev/null
+  ERG="$?"
+  if [ "${ERG}" = "0" ]
+  then
+    echo "${software}"
+  else
+    edition="$(echo "${software}" | awk '{print toupper($0)}')_EDITION"
```

Comment:
this is not the edition but the name of the variable potentially holding the edition...

Reply:
Same as above, and the variable name was chosen more meaningfully